test.py

```diff
@@ -262,30 +262,37 @@ async def add_test_list(self) -> None:
 
         pending = set()
         for shortname in lst:
+            t = os.path.join(self.name, shortname)
```

Comment:
I know that you slightly modified the code, but could you please rename variables to have more sense and better readability? It's related to all modified lines in this file.

Reply:
OK, renamed t in the 2nd patch of the PR not to introduce more churn here